service/key/key.go

```diff
@@ -23,10 +23,18 @@ func AvailabilityZone(customObject awstpr.CustomObject) string {
 	return customObject.Spec.AWS.AZ
 }
 
+func ClusterCustomer(customObject awstpr.CustomObject) string {
+	return customObject.Spec.Cluster.Customer.ID
+}
```

Comment:
ClusterCustomer is a bit confusing because it's called Customer in `clustertpr` but its actually the Organisation in Happa / gsctl. This is used for labeling the k8s namespace and I think the label is useful.